### PR TITLE
[KIALI-3249] [PF4] Move filters to PF4

### PR DESCRIPTION
**Pull request by aljesusg**
Move filters to PF4 
JIRA: [KIALI-3249](https://issues.jboss.org/browse/KIALI-3249)

Screenshot from 2019-08-22 14-35-19
Screenshot from 2019-08-22 14-35-09
Screenshot from 2019-08-22 14-34-50
Screenshot from 2019-08-22 14-34-31

### CODE REVIEW

**Comment by hhovsepy** [comment]
Should the filters for Overview page be on the right side?
Screenshot from 2019-08-23 10-40-59

File: src/pages/AppList/FiltersAndSorts.ts [hunk 1]
```diff
@@ -73,7 +74,7 @@ const appNameFilter: FilterType = {
   id: 'appname',
   title: 'App Name',
   placeholder: 'Filter by App Name',
-  filterType: 'text',
+  filterType: TextInputTypes.text,
```
Comment (xeviknal) [inline src/pages/AppList/FiltersAndSorts.ts]: :clap: :clap:

**Comment by aljesusg** [comment]
> Should the filters for Overview page be on the right side?
> Screenshot from 2019-08-23 10-40-59

mmmm it shouldn't...let me check it

File: src/components/Filters/StatefulFilters.tsx [hunk 2]
```diff
@@ -127,8 +123,10 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
     this.updateActiveFilters(activeFilters);
   };
 
-  selectFilterType = (filterType: FilterType) => {
+  selectFilterType = value => {
```
Comment (xeviknal) [inline src/components/Filters/StatefulFilters.tsx]: This is more a question than a request of a change. Why did you need to remove the type? Is there a way to easily add the type here?

Comment (aljesusg) [reply src/components/Filters/StatefulFilters.tsx]: nah, we can add :string

**Comment by aljesusg** [comment]
@xeviknal, comments addresed, @hhovsepy  fixed toolbar in overview page. Thanks !!!
